tests/pivots/spreadsheet_pivot/spreadsheet_pivot_side_panel.test.ts

```diff
@@ -490,6 +498,36 @@ describe("Spreadsheet pivot side panel", () => {
     expect(mockNotify).toHaveBeenCalledTimes(1);
   });
 
+  test("notification should not be triggered when the pivot opened in the side panel differs from the pivots visible in the viewport.", async () => {
+    const mockNotify = jest.fn();
+    const notificationStore = env.getStore(NotificationStore);
+    notificationStore.updateNotificationCallbacks({
+      notifyUser: mockNotify,
+    });
+    addPivot(
+      model,
+      "B1:B2",
+      {
+        measures: [{ id: "amount:sum", fieldName: "amount", aggregator: "sum" }],
+      },
+      "3"
+    );
+    // insert the first pivot as static pivot in a new empty sheet
+    const sheet2Id = "sheet2";
+    createSheet(model, { sheetId: sheet2Id });
+    activateSheet(model, sheet2Id);
+    selectCell(model, "G1");
```

Comment:
It's not needed to select G1, is it ?
```suggestion
    createSheet(model, { sheetId: sheet2Id, activate: true });
```